src/main/java/co/elastic/support/diagnostics/commands/RunKibanaQueries.java

```diff
@@ -101,13 +125,13 @@ public int runBasicQueries(RestClient client, DiagnosticContext context) throws
     */
     public void getAllPages(RestClient client, List<RestEntry> queries, int perPage, RestEntry action) throws DiagnosticException {
         // get the values needed to the pagination.
-        RestResult res = client.execQuery(String.format("%s?per_page=1", action.getUrl()));
+        RestResult res = client.execQuery(getPageUrl(action, 1, 100));
```

Comment:
Looking at the test codebase lots of these can't even work, because we still test to version 6.5.0 but lots of APIs that we test against only exist in later versions only... These should be fixed in another PR in my opinion and would be too much burden for thi PR.